packages/paste-website/src/pages/form-elements/select/index.mdx

```diff
@@ -0,0 +1,944 @@
+  <ListItem>
+    Provide clear identification of required fields in the label or at the start of a form. If you use the required
+    field indicator, include the form key at the start of the form.
+    <UnorderedList marginBottom="space0">
+      <ListItem>
+        Use the <inlineCode>required</inlineCode> prop to programatically indicate they are required to browsers.
+      </ListItem>
+    </UnorderedList>
+  </ListItem>
+  <ListItem>
+    Include inline error text with the error icon on any field that errors to make it visually clear that the field
+    changed.
+  </ListItem>
+  <ListItem>
+    If the input has assicated help text or error text, include the <inlineCode>aria-describedby</inlineCode> prop on
+    the input. This should match the <inlineCode>id</inlineCode> of the help or error text.
+  </ListItem>
+</UnorderedList>
+
+## Examples
+
+### Base Select
+
+The base select is the input without any supporting elements (e.g., label, help text, etc).
+
+- **Options** — The list of items contained in a select
+- **Selected value** — (Optional) The value that appears in the select on page load. By default, this is the first option in the list.
+
+The select will fill the width of the container it’s placed in.
+
+<LivePreview scope={{Select, Option, FormLabel}} language="jsx">
+  {`<>
+  <FormLabel htmlFor="twilio_product">Product</FormLabel>
+  <Select id="twilio_product" htmlFor="product">
+    <Option value="messaging">SMS</Option>
+    <Option value="numbers">Phone Numbers</Option>
+    <Option value="video">Video</Option>
+  </Select>
+</>`}
+</LivePreview>
+
+### Select
+
+The select component should include the base select, along with supporting elements to compose a select form field that gives the user the context they need to successfully fill it out.
+
+- **Label** — A label should be included for every form field and provide a short title for its associated input.
+- **Required field indicator** — In a form where there are fewer or as many required fields as optional, use a required indicator to show users which fields are required to be filled out.
+- **Optional field indicator** — In a form where there are more required fields than optional fields, use an optional indicator to show users which fields they don’t have to fill out. In general, ask for information only when needed. Consider removing them otherwise.
+- **Help text** — Text that’s placed below the field to help users prevent an error and describe what makes the form field successful.
+
+<Callout>
+  <CalloutTitle as="h4">Hot accessibility tip</CalloutTitle>
+  <CalloutText>
+    Make sure to connect the <inlineCode>FormLabel</inlineCode> to the <inlineCode>Select</inlineCode>. This is done with the <inlineCode>htmlFor</inlineCode> prop on the label, and the <inlineCode>id</inlineCode> prop on the select. Those two need to match.
+  </CalloutText>
+</Callout>
+
+<LivePreview scope={{Select, Option, FormHelpText, FormLabel}} language="jsx">
+  {`<>
```

Comment:
Can we remove <>, and add "Select a Product" below instead of product (do this for other options too)

Reply:
AFAIK since there are two components on the same level in the return statement (`FormLabel` and `Select`), the `</>` is required to wrap.